### DISCUSSION TITLE
The Magic of Small Databases

### DISCUSSION THREAD

**Discussion by davidgasquez** [datasets/commons#381] [General]
Found [an awesome blog post about building/curating small databases](https://tomcritchlow.com/2023/01/27/small-databases/). I think it'll resonate with people here.

I know @rufuspollock has been thinking about this for a while (Data Pinterest). The post mentions a similar analogy with Substack in the _Market-making Small Databases_ section. 

> Imagine a Substack for databases - an easy tool for creating, maintaining and publishing databases with the ability to restrict parts or all of it behind a pay wall. Pair it with the ability to send email updates to your audience about changes and additions. Maybe even roll in some of the data-cleanup and data-collection magic that machine learning and web scraping offers.

I'd love to explore with you all how a community around this idea could be built and if there is room for a platform to help with this endeavour!

Edit: linking also [a related Twitter thread](https://twitter.com/marcua/status/1619473937041354754) with an [interesting conversation](https://twitter.com/marcua/status/1619905371560378368).

> There have been several well intentioned companies that made platforms for social sharing of data that moved toward commercial versions after the first year or two.

> There's a tension between having individuals/groups working to develop databases according to their own interests & goals and making a product out of that.

> Each repository on GitHub affords different community standards, there’s room for tools that makes it easier for collaborators to share and curate datasets while allowing different practices.

> there’s so much $ to be made quickly in enterprise data that wonderful previous attempts [data.world and Enigma] have been gobbled up. Perhaps the best home/incubator for this is a side project or academia until it’s proven out

**Comment by davidgasquez** [comment]
In the article, there is a "[Small Indexes](https://airtable.com/shrYY94GrqVB4HUsi/tblHPrdomiPbLpod6/viwxizssDJMsGqhg9?backgroundColor=green&blocks=hide)" airtable which led me to find an awesome resource: [Data is Plural](https://www.data-is-plural.com/), an newsletter covering interesting datasets.

The [Data is Plural historical archive](https://docs.google.com/spreadsheets/d/1wZhPLMCHKJvwOkP4juclhjFgqIY8fQFMemwKL2c64vk/edit) contains lots of datasets that might be interesting to index/curate.

**Comment by rufuspollock** [comment]
@davidgasquez great find David and really resonate.

Related but a bit distinct: https://blog.okfn.org/2013/04/22/forget-big-data-small-data-is-the-real-revolution/

PS: I wonder if we should link blog on datahub.io where we just post up interesting links with (optionally) a bit of commentary. This material would be perfect.
